module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -2238,9 +2265,9 @@ protected function setUpAuthenticationFromRequest()
                 $this->params()->fromPost('auth_method')
             )
         );
+
         if (!empty($method)) {
-            $this->getAuthManager()->setAuthMethod($method);
-        }
+        }    $this->getAuthManager()->setAuthMethod($method);
```

Comment:
There seems to be a formatting problem here -- did this brace get moved up a line somehow?